bot/exts/filters/filtering.py

```diff
@@ -481,10 +481,11 @@ async def _has_urls(self, text: str) -> Tuple[bool, Optional[str]]:
         domain_blacklist = self._get_filterlist_items("domain_name", allowed=False)
         for match in URL_RE.finditer(text):
             for url in domain_blacklist:
-                blacklisted_parsed = tldextract.extract(url)
-                url_parsed = tldextract.extract(match.group(1).lower())
-                if blacklisted_parsed.registered_domain == url_parsed.registered_domain:
-                    return True, self._get_filterlist_value("domain_name", url, allowed=False)["comment"]
+                if url.lower() in match.group(1).lower():
+                    blacklisted_parsed = tldextract.extract(url.lower())
+                    url_parsed = tldextract.extract(match.group(1).lower())
```

Comment:
L486 can be before L483

Reply:
i dont see the need of creating the extra variable unless we know we'll need it.